src/frontend/components/PublicVideoLiveJitsi/index.spec.tsx

```diff
@@ -94,9 +98,44 @@ describe('<PublicVideoLiveJitsi />', () => {
       name: /Leave discussion/i,
     });
     screen.getByRole('button', {
-      name: /Chat/i,
+      name: /Show chat/i,
     });
 
     await imageSnapshot(1920, 1080);
   });
+
+  it('renders jitsi, live info, chat and actions on small screen', async () => {
+    useParticipantWorkflow.setState({
+      accepted: true,
+    });
+
+    render(
+      wrapInIntlProvider(
+        wrapInRouter(
+          <Grommet theme={theme} style={{ height: '100%' }}>
+            <ResponsiveContext.Provider value="small">
+              <PublicVideoLiveJitsi video={mockVideo} />
+              <GlobalStyles />
+            </ResponsiveContext.Provider>
+          </Grommet>,
+        ),
+      ),
+    );
+
+    expect(mockJitsi).toHaveBeenCalled();
+
+    //  expect chat input to be present
+    expect(screen.getByText('Join the chat')).not.toBeVisible();
```

Comment:
You should maybe change the comment, it is confusing. Should be present but not visible ?

Reply:
816472c